src/styles/variables.scss

```diff
@@ -0,0 +1,17 @@
+// colors
```

Comment:
we need `.module.scss` to get access from js: `import variables from '@/styles/variables.module;`